dev/com.ibm.ws.security.social/src/com/ibm/ws/security/social/internal/utils/OAuthClientHttpUtil.java

```diff
@@ -256,119 +236,22 @@ void setAuthorizationHeader(String baUsername, @Sensitive String baPassword, Str
         }
     }
 
-    void debugPostToEndPoint(String url, @Sensitive List<NameValuePair> params, String baUsername, @Sensitive String baPassword, String accessToken, final List<NameValuePair> commonHeaders) {
-        if (!tc.isDebugEnabled()) {
-            // Trace isn't enabled, so don't bother executing the method
-            return;
-        }
-
-        // Trace the cURL command that will be used using the provided arguments
-
-        Tr.debug(tc, "postToEndpoint: url: " + url + " headers: " + commonHeaders + " params: " + "*****" + " baUsername: " + baUsername + " baPassword: " + (baPassword != null ? "****" : null) + " accessToken: " + accessToken);
-        StringBuffer sb = new StringBuffer();
-        sb.append("curl -k -v");
-        if (commonHeaders != null) {
-            for (Iterator<NameValuePair> i = commonHeaders.iterator(); i.hasNext();) {
-                NameValuePair nvp = i.next();
-                sb.append(" -H \"");
-                sb.append(nvp.getName());
-                sb.append(": ");
-                sb.append(nvp.getValue());
-                sb.append("\"");
-            }
-        }
-        if (params != null && params.size() > 0) {
-            sb.append(" -d \"");
-            for (Iterator<NameValuePair> i = params.iterator(); i.hasNext();) {
-                NameValuePair nvp = i.next();
-                String name = nvp.getName();
-                sb.append(name);
-                sb.append("=");
-                if (name.equals("client_secret")) {
-                    sb.append("*****");
-                } else {
-                    sb.append(nvp.getValue());
-                }
-
-                if (i.hasNext()) {
-                    sb.append("&");
-                }
-            }
-            sb.append("\"");
-        }
-        if (baUsername != null && baPassword != null) {
-            sb.append(" -u \"");
-            sb.append(baUsername);
-            sb.append(":");
-            sb.append("****");
-            sb.append("\"");
-        }
-        if (accessToken != null) {
-            sb.append(" -H \"Authorization: bearer ");
-            sb.append(accessToken);
-            sb.append("\"");
-        }
-        sb.append(" ");
-        sb.append(url);
-
-        Tr.debug(tc, "CURL Command: " + sb.toString());
+    void debugPostToEndPoint(String url,
+            @Sensitive List<NameValuePair> params,
+            String baUsername,
+            @Sensitive String baPassword,
+            String accessToken,
+            final List<NameValuePair> commonHeaders) {
+        httpUtils.debugPostToEndPoint(url, params, baUsername, baPassword, accessToken, commonHeaders);
```

Comment:
I'd probably look at taking out methods like this that are now just wrappers to `HttpUtils` methods.